rsrc/linux/com.github.Murmele.Gittyup.desktop

```
@@ -2,11 +2,11 @@
 Encoding=UTF-8
 Name=Gittyup
 Comment=Graphical Git client for Windows, Linux and macOS
-Exec=Gittyup
+Exec=gittyup
 StartupWMClass=gittyup
 Icon=gittyup
 Terminal=false
 Type=Application
 Categories=Development;
 X-Desktop-File-Install-Version=0.1
```

What is this key supposed to do? And is in really necessary? From what I know of the desktop file spec I would have just done this instead:

```suggestion
Version=1.0
```

If there is some good reason for the X-... key and version info that's fine, but I suspect that was just copied from somewhere and it actually doesn't make any sense here.